### ISSUE TITLE
How can I setup or change keybindings?

### ISSUE THREAD

**Issue by Andras-Csanyi** [seblj/roslyn.nvim#24]
Hi all,

this is a great work here! Thanks for it!
How can I set up keybindings? For example `gd` select all occurrences of the string while with other plugins (java) it goes to definition.

**Comment by seblj** [comment]
There isn't any custom keybindings for this specifically. Must be something in your config that isn't configured correctly.

`vim.lsp.buf.definition` will go to definition just like normal

**Comment by konstantin-mohin** [comment]
@Andras-Csanyi  have the same issue all lsp shortcuts stop working , have you succeed ?

**Comment by seblj** [comment]
Without seeing your config or anything, my initial thought is thst you are setting then in an `on_attach` callback.

If you just use `:help LspAttach` instead, it should just work like normal

**Comment by Andras-Csanyi** [comment]
Sorry guys, I'll redo my config and we will see.

**Comment by Andras-Csanyi** [comment]
So the phenomena I have is the following, and I believe it is a config issue. However, I don't know how this plugin gets the lsp client config.

What is happening:
- at any language (including java) when there is a squiggly line and in normal mode I press "gl" a popup comes up and shows the message
- in C# this doesn't work, and other LSP specific keymaps, others work just fine

My config is located [here](https://github.com/Andras-Csanyi/configs/tree/main/.config/nvim) and based on [this](https://youtube.com/playlist?list=PLhoH5vyxr6Qq41NFL4GvhFp-WLd5xzIzZ&si=Lj5hYKE5J9OxZdte) series from Chris@Machine.

The LSP is configured [here](https://github.com/Andras-Csanyi/configs/blob/main/.config/nvim/lua/andrascsanyi/lsp/handlers.lua).

I'd like to show the java config, since it is complicated a bit and custom. When java, JDTLS, is configured it gets the client [here](https://github.com/Andras-Csanyi/configs/blob/main/.config/nvim/lua/andrascsanyi/ftplugin/java.lua#L110).

So, it seems to be a configuration issue.

I'll look at the `:help LspAttach` and check how `LunarVim` does it and get back to you guys.

**Comment by seblj** [comment]
Yes it seems to be as I expected. You can replace the entire `on_attach` method and replace it with an autocmd with `LspAttach` as the event.

**Comment by Andras-Csanyi** [comment]
Oh, I didn't know your config is available. Thanks for the help!
